Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Build Stage
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
+
+## Install build dependencies.
+RUN apt-get update 
+RUN apt-get -y install g++ git libelf-dev libcap-dev apt-transport-https curl gnupg 
+
+RUN curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
+RUN mv bazel.gpg /etc/apt/trusted.gpg.d/
+RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+RUN apt update && apt install -y bazel
+
+ADD . /mayhem-perf_data_converter
+WORKDIR /mayhem-perf_data_converter
+RUN bazel build src:perf_to_profile
+RUN cp /mayhem-perf_data_converter/bazel-bin/src/perf_to_profile /
```

Comment:
Perhaps you meant for this to be done as a package stage? Are you able to split this into multiple stages in the Dockerfile to reduce the image size?

Reply:
It's not a hard requirement, but it's better if you can implement multi-stage builds. We'll eventually have at least 1000 jobs running under mayhemheroes, and pushing and pulling that many images can take a while; keeping image size down is a best practice in general, but also for our purposes.

Reply:
Sounds good, I'll try to do so for all my submissions; please forgive me if I run behind your initial PR reviews!